SDWebImage/SDImageCacheConfig.h

```diff
@@ -68,13 +68,14 @@ typedef NS_ENUM(NSUInteger, SDImageCacheConfigExpireType) {
 @property (assign, nonatomic) NSDataWritingOptions diskCacheWritingOptions;
 
 /**
- * The maximum length of time to keep an image in the cache, in seconds.
+ * The maximum length of time to keep an image in the disk cache, in seconds.
+ * Setting this to a negative value means no expiring.
```

Comment:
Add `Setting this to zero means expiring immediately` comment here to specify the detail behavior.